contrib_bots/lib/myanimelist/mal.py

```diff
@@ -0,0 +1,69 @@
+import requests
+from bs4 import BeautifulSoup
+import re
+# See readme.md for instructions on running this code.
+malSearchUrl = "https://myanimelist.net/search/all?q="
```

Comment:
By convention Python constants should be in capital letters and words should be separated by underscores:
https://www.python.org/dev/peps/pep-0008/#constants